tests/unit/test_serialization.py

```diff
@@ -202,7 +202,8 @@ def test_flatten_unions_map() -> None:
     )
     record = {"foo": {"attr1": {"string": "sample data"}}}
     flatten_record = {"foo": {"attr1": "sample data"}}
-    assert flatten_unions(typed_schema.schema, record) == flatten_record
+    a = flatten_unions(typed_schema.schema, record)
```

Comment:
Could this variable have a more descriptive name (or it could be removed like how it previously was)?

Reply:
I will remove it from tests.